### ISSUE TITLE
Support TLDs

### ISSUE THREAD

**Issue by volosied** [omnifaces/vdldoc#16]
Hello,

Due to the Javax-Jakarta namespace transition, JSTL-API project needs to be updated, particularly around TLDs. These old TLDs need to have new URIs and namespace schema definitions.  See https://github.com/eclipse-ee4j/jstl-api/issues/144

However, the project currently uses the ancient 'taglibrarydoc:tlddoc:1.3' implementation which provides many incompatibilities with the newer EE9+ namespaces and it references 'javax' when generating the docs.  

The JSTL-API community would like to move away from this old library and use vdldoc instead. However, I see it doesn't support TLDs. 

In a nutshell, would it be possible to have vdldoc code updated to support TLD files formats? 

Thank you

**Comment by BalusC** [comment]
That's going to be really a lot of work because vdldoc has been rewritten specifically for VDL file format.

It'll be easier to fork/update the tlddoc itself.

**Comment by volosied** [comment]
Thanks for the quick response! I'll look into updating tlddoc then.
